constants.go

```diff
@@ -317,9 +318,6 @@ const (
 	// LinuxOS is the GOOS constant used for Linux.
 	LinuxOS = "linux"
 
-	// WindowsOS is the GOOS constant used for Microsoft Windows.
-	WindowsOS = "windows"
-
```

Comment:
Is this intentional? This seems to be a better location for the constant.

Reply:
Yes, any constant needed by the `api` package needs to be in the `api` package. Having two constants files is not ideal, and we definitely need to find a better solution, so I'll think about this for 7.0.

Reply:
Understood. Wouldn't it then however make more sense to move the OS constants all together, i.e. to move `LinuxOS` & `DarwinOS` alongside with `WindowsOS` instead of having them split up?